src/styles/blocks/button.scss

```diff
@@ -0,0 +1,40 @@
+.button {
+  box-sizing: border-box;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  height: 56px;
+  border: 2px solid $color-blue;
+  border-radius: 8px;
+
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 14px;
+  text-decoration: none;
+  color: $color-blue;
+
+  transition: all 0.3s;
```

Comment:
don't use transition all. Indicate which properties you want to use transition for